cpp/json_ffi/json_ffi_engine.cc

```diff
@@ -128,12 +129,13 @@ class JSONFFIEngineImpl : public JSONFFIEngine, public ModuleNode {
       ICHECK_EQ(args.size(), 1);
       Array<RequestStreamOutput> delta_outputs = args[0];
       Array<String> responses = this->GetResponseFromStreamOutput(delta_outputs);
-      this->request_stream_callback_(responses);
+      for (const String response : responses) {
+        this->request_stream_callback_(static_cast<std::string>(response));
```

Comment:
instead of stream it back one at a time, how about we construct a json array and stream them all at once